docs/src/pages/components/modal.mdx

```diff
@@ -188,10 +191,53 @@ And voila! Here is our `PageExample` component rendered with our `ModalAndButton
 
 <br /><br />
 
+## Custom Background
+
+If there is a need to have a custom background for the modal, it can be accomplished by sending an
+uninstantiated `styled-component` to the `ModalProvider` as a prop.
+
+```jsx
+import { BaseModalBackground, ModalProvider } from '@retailmenot/anchor';
+import styled from '@xstyled/styled-components';
+
+const CustomBackground = styled(BaseModalBackground)`
+    padding: 0;
```

Comment:
In this particular case, we do actually have a mechanism via props to accomplish this:

```jsx
<Modal backgroundProps={{ padding: '0', opacity: 1 }}>
  ...
</Modal>
```

Though `padding` and `opacity` are the only `backgroundProps` props we currently support

Reply:
I'll add this info in.